test/e2e/network_test.go

```diff
@@ -144,6 +145,35 @@ func testEastWestNetworkingAcrossWindowsNodes(t *testing.T) {
 	}
 }
 
+// testEastWestNetworkingMetrics deploys Linux pod and tests that it can communicate with Windows node's metrics port
+func testEastWestNetworkingMetrics(t *testing.T) {
```

Comment:
I suggest calling this `testMetrics`. Is there a way to check for the expected metrics?